README.md

```diff
@@ -82,13 +82,13 @@ The following table lists the configurable parameters of the Verdaccio chart,
 and their default values.
 
 | Parameter                            | Description                                                                                                       | Default                        | Reference                                                                                                                 |
-| ------------------------------------ | ----------------------------------------------------------------------------------------------------------------- | ------------------------------ | ------------------------------------------------------------------------------------------------------------------------- |
+| ------------------------------------ | ----------------------------------------------------------------------------------------------------------------- |--------------------------------| ------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
What is the purpose of this change?

Reply:
Updated table with version of Verdaccio supported 😄

Reply:
No, I am talking about this line in particular. You basically changed `| --...-- |` to `|---...---|` in one of the columns 🤔

Reply:
Oh, I'll fix that right away